pipeline-runner/codecov.yaml

```diff
@@ -14,6 +14,4 @@ coverage:
         # this allows a 5% drop from the previous base commit coverage
         threshold: 3%
 
-    patch:                    # measuring the coverage of new changes
-      default:
-        enabled: no
+    patch: off                   # measuring the coverage of new changes
```

Comment:
why off?

Reply:
I was just trying out if turning the value to `off` disables the patch detection. It doesn't seem so. Codecov seems to still carry out patch changes for this PR, even though it's set to be off (and it is actually set to be off for this repo). This doesn't seem happen with other PRs

Reply:
There's an issue about this here : https://community.codecov.com/t/cannot-disable-codecov-patch-check/682/45